src/views/addProperty.js

```diff
@@ -132,6 +135,19 @@ export class AddProperty extends Component {
                                             />
                                             {errors.zipcode ? (<div className="error-message">{errors.zipcode}</div>) : null}
                                         </div>
+                                        <div className="form-row columns">
+                                            <label className="column is-one-fifth" htmlFor="units">Units</label>
+                                            <Field
+                                                className="column form-field"
+                                                type="text"
+                                                name="units"
+                                                onChange={handleChange}
+                                                value={values.units}
+                                                placeholder="Number of units"
+                                                error={errors.units}
+                                            />
+                                            {errors.units ? (<div className="error-message">{errors.units}</div>) : null}
+                                        </div>
                                         {/* This field needs to be included after
                                             the units parameter is added to the api endpoint */}
                                         {/* <div className="form-row columns">
```

Comment:
Looks like there's an older commented out version of this field right here so we can get rid of it 😄

Reply:
LOL that's pretty funny. Fixing this now.

Reply:
Give it one more shot.